apps/frontend/core-dapp/src/features/position/PositionLabel.tsx

```diff
@@ -9,8 +9,8 @@ type Props = {
 
 const Wrapper = styled.div<Props>`
   color: ${({ theme, positionType }): string => theme.positionType[positionType]};
-  font-size: ${({ theme }): string => theme.fontSize.xs};
-  font-weight: ${({ theme }): number => theme.fontWeight.medium};
+  font-size: ${({ theme }): string => theme.fontSize.md};
```

Comment:
Should this be sm (14px) instead of md?
```suggestion
  font-size: ${({ theme }): string => theme.fontSize.sm};
```